snuba/utils/streams/backends/local/storages/file.py

```diff
@@ -30,7 +31,7 @@ class PickleCodec(Codec[bytes, Tuple[TPayload, datetime]]):
     def encode(self, value: Tuple[TPayload, datetime]) -> bytes:
         return pickle.dumps(value)
 
-    def decode(self, value: bytes) -> Tuple[TPayload, datetime]:
+    def decode(self, value: bytes) -> Any:
```

Comment:
Maybe this should assert instead of changing to Any? The nice thing about these generics is that they guarantee the typing.

Reply:
you cannot assert that an object is of the generic type. Those types do not exist at runtime.